packages/manager/apps/hycu/src/pages/dashboard/general-information/BillingInformations/BillingInformationsTile.tsx

```diff
@@ -100,17 +107,31 @@ const BillingInformationsTile = ({ serviceName }: { serviceName: string }) => {
           value: (
             <div className="flex flex-col gap-4">
               <div>
-                {isLoading
-                  ? Array.from({ length: 3 }).map((_, index) => (
-                      <OsdsSkeleton key={index} />
-                    ))
-                  : serviceDetails?.data.customer.contacts.map((contact) => (
+                {isLoading ? (
+                  Array.from({ length: 3 }).map((_, index) => (
+                    <OsdsSkeleton key={index} />
+                  ))
+                ) : (
+                  <>
+                    {serviceDetails?.data.customer.contacts.map((contact) => (
                       <Description key={contact.type}>{`${
                         contact.customerCode
                       } ${t(
                         `hycu_dashboard_contact_type_${contact.type}`,
                       )}`}</Description>
                     ))}
+                    {isLoadingContactUrl ? (
+                      <OsdsSkeleton />
+                    ) : (
+                      <Links
+                        href={(contactUrl as string) ?? '#'}
+                        type={LinkType.next}
+                        label={t('hycu_dashboard_field_label_manage_contacts')}
+                        className="mt-4"
+                      />
+                    )}
```

Comment:
Just little suggestion with a question : 

When I implemented this I use disabled because the content not change and the load is very fast (Is not a web request just a local async function)

So I find this more readable, more maintainable and have better UX / CLS

What do you think ?

But is good implementation too

Reply:
+1

Reply:
As seen together, we decided to display a skeleton while loading the URL, independently of the main isLoading.
When migrating to ODS18, we'll have to remember to update the component to add native attributes like `disabled` (not available yet), and replace `OsdsLink` with the `Links`component from MRC